### PR TITLE
routing docs

### PR DESCRIPTION
**Pull request by alexandra-plana**
A mess tbh. 
to access  all docs = '/documentation/project/:project_id',
i have to link the click in the dashboard to this information.

The MainDocs page has access to the info for both project docs and milestone docs
Its a matter of figuring out the routes.

Also to create a doc we need to link it with a milestone since that is the relationship in the tables.
Trying to figure out the logic for that too since it will be a required field when creating a milestone.

I'm truly sorry

i've created the query for all docs and cleaned it up

### CODE REVIEW

**Comment by alexandra-plana** [comment]
We can access the milestone docs through '/documentation/:milestone_id'
which is how i was linking the doc to the milestone

File: client/common/components/Article.tsx [hunk 1]
```diff
@@ -70,6 +86,9 @@ export default function Article({ data }: any) {
   };
   const handleBookmark = () => {
     console.log('ive been bookmarked but i have no functionality whatsoever');
+    console.log('article id', data.id);
+    console.log('user', reduxCurrentUser);
+    bookmarkArticle(data.id, reduxCurrentUser); // article , user
   };
   return (
```
Comment (lexarisa) [inline client/common/components/Article.tsx:93]: so much going on in this file!! <3

**Review by lexarisa — APPROVED** [review]
Impressive work!!! thank you so muchhh <3 <3

**Review by thaistcosta — APPROVED** [review]
Thank youuu 🍪  !!!!!! AMAZIIIING !!!!

SO....


You have a Documentation page that holds a Main docs Component... when the person is being redirect to your Documentation page your page is going to receive all the info it needs ... we can build optional queries for the same page
❤️

**Pull request merged**